### ISSUE TITLE
Add shortcut.mojo to the mojo profile

### ISSUE THREAD

**Issue by afandria** [vanadium/issues#992]
`jiri-v23-profile/mojo/mojo.go` needs to add `shortcut.mojo` to the list of Android service names. This is used to place shortcuts for Android Mojo apps.

Before we do this, there was something problematic about `shortcut.mojo` in the latest version of Mojo. @aghassemi, do you remember what it was?

@nlacasse @bprosnitz

**Comment by aghassemi** [comment]
yes latest one does not work. I am using the one from a while back that I got from Alex. I have published it at http://storage.googleapis.com/mojo_services/syncslides/shortcut.mojo for now

**Comment by afandria** [comment]
This has been here since mojo profile v4, but I need to actually verify that it can be used. (In any case, I should also update mojo.shared to have the shortcut making rule. #942)
